README.rst

```diff
@@ -19,36 +19,34 @@
 .. |Binder| image:: https://mybinder.org/badge_logo.svg
    :target: https://mybinder.org/v2/gh/sunpy/sunpy/main?filepath=examples
 
-SunPy is an open-source Python library for Solar Physics data analysis and visualization.
-Our homepage `SunPy`_ has more information about the project.
+``sunpy`` is an open-source Python library for Solar Physics data analysis and visualization.
+`Our homepage has more information about the SunPy project. <https://sunpy.org>`_
 
-For some examples of using SunPy see our `gallery`_, to see the latest changes in SunPy see our `Changelog`_.
+For some examples of using ``sunpy`` see our `gallery`_, to see the latest changes in ``sunpy`` see our `Changelog`_.
```

Comment:
Make these two sentences instead of a comma splice, especially given how unrelated the thoughts are.

Reply:
Fixed hopefully.